src/components/Header/Contacts/Contacts.tsx

```diff
@@ -1,22 +1,40 @@
-import React from 'react'
+import React, { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
 
+import { useToggle } from 'hooks/toggleHook'
 import adressIcon from 'img/svg/adress-icon.svg'
 import callIcon from 'img/svg/call-icon.svg'
 import facebookIcon from 'img/svg/facebook-icon.svg'
 import instagramIcon from 'img/svg/instagram-icon.svg'
 import timeIcon from 'img/svg/time-icon.svg'
 
+import { ModalWindow } from 'components/ModalWindow'
+
 import styles from './Contacts.module.scss'
 
 export const Contacts = () => {
+    const { isOpen: isOpenModalLanguage, onToggle: onToggleModalLanguage } = useToggle()
+
+    const { t, i18n } = useTranslation()
+
+    const lngs = ['en', 'ua']
+
+    const onToggleModal = useCallback(() =>
```

Comment:
why do we need this function if we have the same function inside?

Reply:
fixed